src/main/java/net/wurstclient/commands/TpCmd.java

```diff
@@ -34,7 +41,7 @@ public void call(String[] args) throws CmdException
 		BlockPos pos = argsToPos(args);
 		
 		ClientPlayerEntity player = MC.player;
-		player.setPosition(pos.getX() + 0.5, pos.getY(), pos.getZ() + 0.5);
+		player.setPosition(pos.getX() + 0.5, pos.getY()+0.0, pos.getZ() + 0.5);

```

Comment:
What does adding 0.0 do? Just curious.

Reply:
Idk, maybe trash from debug

Reply:
Have you tried removing it and seeing if it works without the addition? I think extra redundant code makes it harder to understand.